tools/gulp/constants.ts

```diff
@@ -29,6 +29,12 @@ export const HTML_MINIFIER_OPTIONS = {
   removeAttributeQuotes: false
 };
 
+/** Packages that will be published to NPM by the release task. */
+export const RELEASE_PACKAGES = [
+  'cdk',
+  'material',
+];
```

Comment:
Ah, you mean it to be like a configuration for a re-usable library. That makes more sense. In that case I'd be okay with it if we rename the file "build-config.ts" for now.